CMakeLists.txt

```diff
@@ -169,32 +169,46 @@ foreach(HEADER_SRCPATH IN ITEMS ${AWS_COMMON_HEADERS} ${AWS_COMMON_OS_HEADERS} $
 		message(ERROR "Couldn't find source root for header ${HEADER_SRCPATH}")
 	endif()
 
-    install(FILES ${HEADER_SRCPATH} DESTINATION ${HEADER_DSTDIR})
+    install(FILES ${HEADER_SRCPATH}
+            DESTINATION ${HEADER_DSTDIR}
+            COMPONENT Development)
 endforeach()
 
-set(LIBRARY_DIRECTORY ${CMAKE_INSTALL_PREFIX}/lib)
+set(LIBRARY_DIRECTORY lib)
 # Set the default lib installation path on GNU systems with GNUInstallDirs
 if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT AND UNIX AND NOT APPLE)
```

Comment:
Along the same lines, I think `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT` would be unneeded, as you would still want the libraries to go to `lib64` (or equivalent) under a local install prefix.